crates/ruff/src/rules/pycodestyle/rules/too_many_blank_lines.rs

```diff
@@ -0,0 +1,96 @@
+use ruff_python_ast::source_code::Locator;
+use ruff_text_size::{TextRange, TextSize};
+
+use ruff_diagnostics::{AlwaysAutofixableViolation, Diagnostic, Edit, Fix};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::newlines::Line;
+
+/// ## What it does
+/// Checks for too many (>=3) blank lines.
+///
+/// ## Why is this bad?
+/// PEP 8 recommends the using blank lines as following:
+/// - Two blank lines are expected between functions and classes
+/// - One blank line is expected between methods of a class.
+///
+/// ## Example
+/// ```python
+/// def func1():
+///     pass
+///
+///
+///
+/// def func2():
+///     pass
+/// ```
```

Comment:
Because this issue is fixed by black, the rule name should be added to the [`KNOWN_FORMATTING_VIOLATIONS` list](https://github.com/charliermarsh/ruff/blob/main/scripts/check_docs_formatted.py#L25-L54) to stop CI flagging that it should be fixed.

Reply:
Is there a list somewhere with black's rules names ?

Reply:
In this case, you would add `too_many_blank_lines` to the list as `KNOWN_FORMATTING_VIOLATIONS` relates to Ruff's rules rather than black.

Reply:
I see, thanks!